Formula/pythran.rb

```diff
@@ -0,0 +1,110 @@
+class Pythran < Formula
+  desc "Ahead of Time compiler for numeric kernels"
+  homepage "https://pythran.readthedocs.io/"
+  url "https://files.pythonhosted.org/packages/2e/e9/bc3c16d683dbba5d34666b9861d1c1fdb07da0539655a3cf23a62e4cafa1/pythran-0.9.11.tar.gz"
+  sha256 "a317f91e2aade9f6550dc3bf40b5caeb45b7e012daf27e2b3e4ad928edb01667"
+  license "BSD-3-Clause"
+  head "https://github.com/serge-sans-paille/pythran.git"
+
+  keg_only <<~EOS
+    this formula is mainly used internally by other formulae.
+    Users are advised to use `pip` to install pythran
+  EOS
+
+  depends_on "gcc" # for OpenMP
+  depends_on "numpy"
+  depends_on "openblas"
+  depends_on "python@3.9"
+  depends_on "six"
+
+  resource "beniget" do
+    url "https://files.pythonhosted.org/packages/50/b9/7d864a50d5d2821686d4374400de45c8e0678e553657343f5f0ea6aff98d/beniget-0.3.0.tar.gz"
+    sha256 "062c893be9cdf87c3144fb15041cce4d81c67107c1591952cd45fdce789a0ff1"
+  end
+
+  resource "decorator" do
+    url "https://files.pythonhosted.org/packages/da/93/84fa12f2dc341f8cf5f022ee09e109961055749df2d0c75c5f98746cfe6c/decorator-4.4.2.tar.gz"
+    sha256 "e3a62f0520172440ca0dcc823749319382e377f37f140a0b99ef45fecb84bfe7"
+  end
+
+  resource "gast" do
+    url "https://files.pythonhosted.org/packages/83/4a/07c7e59cef23fb147454663c3271c21da68ba2ab141427c20548ae5a8a4d/gast-0.4.0.tar.gz"
+    sha256 "40feb7b8b8434785585ab224d1568b857edb18297e5a3047f1ba012bc83b42c1"
+  end
+
+  resource "networkx" do
+    url "https://files.pythonhosted.org/packages/b0/21/adfbf6168631e28577e4af9eb9f26d75fe72b2bb1d33762a5f2c425e6c2a/networkx-2.5.1.tar.gz"
+    sha256 "109cd585cac41297f71103c3c42ac6ef7379f29788eb54cb751be5a663bb235a"
+  end
+
+  resource "ply" do
+    url "https://files.pythonhosted.org/packages/e5/69/882ee5c9d017149285cab114ebeab373308ef0f874fcdac9beb90e0ac4da/ply-3.11.tar.gz"
+    sha256 "00c7c1aaa88358b9c765b6d3000c6eec0ba42abca5351b095321aef446081da3"
+  end
+
+  def install
+    on_macos do
+      gcc_major_ver = Formula["gcc"].any_installed_version.major
+      inreplace "pythran/pythran-darwin.cfg" do |s|
+        s.gsub!(/^include_dirs=/, "include_dirs=#{Formula["openblas"].opt_include}")
+        s.gsub!(/^library_dirs=/, "library_dirs=#{Formula["openblas"].opt_lib}")
+        s.gsub!(/^blas=.*/, "blas=openblas")
```

Comment:
Can we check that this links correctly with `openblas`, and not, say Apple Accelerate instead?

Reply:
I haven't seen it ever try to link Apple Accelerate. With current config, it links `openblas` and `gcc` libs:
```console
❯ otool -L arc_distance.cpython-39-darwin.so
arc_distance.cpython-39-darwin.so:
	/usr/local/opt/openblas/lib/libopenblas.0.dylib (compatibility version 0.0.0, current version 0.0.0)
	/usr/local/opt/gcc/lib/gcc/11/libstdc++.6.dylib (compatibility version 7.0.0, current version 7.29.0)
	/usr/lib/libSystem.B.dylib (compatibility version 1.0.0, current version 1292.100.5)
	/usr/local/lib/gcc/11/libgcc_s.1.dylib (compatibility version 1.0.0, current version 1.0.0)
```

If I create a local config `~/.pythranrc` that reverts all configs to default, then I get:
```console
❯ otool -L arc_distance.cpython-39-darwin.so
arc_distance.cpython-39-darwin.so:
	/usr/lib/libc++.1.dylib (compatibility version 1.0.0, current version 905.6.0)
	/usr/lib/libSystem.B.dylib (compatibility version 1.0.0, current version 1292.100.5)
```